src/components/RadioButton/index.js

```diff
@@ -0,0 +1,72 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['change', 'value:change', 'text:change', 'name:change'];
+
+const html = `
+<label class="radio-container">
+    <input data-select="radio-button" type="radio" name="" value="">
+    <span data-select="radio-button-text" class="radio-dot"></span>
+</label>
+`;
+
+export default function RadioButton({
+  isChecked = false,
+  text = '',
+  value = '',
+  name = '',
+} = {}) {
```

Comment:
Como vamos adicionar o método de desabilitar, precisamos da inicialização dele
```suggestion
export default function RadioButton({
  isChecked = false,
  text = '',
  value = '',
  name = '',
  isDisabled = false,
} = {}) {
```